### DISCUSSION TITLE
how to do that the cursor follows focus

### DISCUSSION THREAD

**Discussion by vovahurash** [i3/i3#6296] [Q&A]
someone have the script or know how to make the cursor to follow focus? like when i'm changing my focus using mod+right arrow, my cursor moves to window that i'll be focused on
edit: like in hyprland

**Comment by okraits** [comment]
I guess you can implement that with a script which subscribes to the window focus event via IPC (https://i3wm.org/docs/ipc.html#_window_event) and then moves the cursor with xdotool.

**Reply by vovahurash** [reply]
my problem is.. i don't know how to write scripts and how to write this exactly script that i want
if you have this script written, or you can write it, i'll be happy if you send it to me
maybe if no one can help me, then i will learn myself how to do this
